### PR TITLE
Qtbase build script

### PR DESCRIPTION
**Pull request by antonmolodykh**
Script for building qtbase

### CODE REVIEW

File: build_qtbase.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -x
+
+sudo sed -i -e 's/#deb-src/deb-src/' /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get -y build-dep libqt5gui5
+sudo apt-get -y build-dep qt4-x11
+sudo apt-get -y install libudev-dev libinput-dev libts-dev libxcb-xinerama0-dev libxcb-xinerama0
+
+INIT_FOLDER=$(pwd)
+read -p "\nPress 'y' if you want to start the compilation from scratch or 'N' to try to use the already existing things (y/N)" -n 1 -r -s REBOOT && echo
```
Comment (vpetersson) [inline build_qtbase.sh]: Why don't you just do a conditional check to see if the folder exist?

File: build_qtbase.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -x
+
+sudo sed -i -e 's/#deb-src/deb-src/' /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get -y build-dep libqt5gui5
+sudo apt-get -y build-dep qt4-x11
+sudo apt-get -y install libudev-dev libinput-dev libts-dev libxcb-xinerama0-dev libxcb-xinerama0
+
+QT_FOLDER=/home/pi/src/qt5
+INIT_FOLDER=$(pwd)
```
Comment (vpetersson) [inline build_qtbase.sh]: Remove

File: build_qtbase.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  cd qtbase
+  ./configure -release -opengl es2 -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/usr/bin/ -opensource -confirm-license -release -make libs -prefix /usr/local/qt5pi -extprefix $QT_FOLDER/qt5pi -no-use-gold-linker
+else
+  cd $QT_FOLDER/qtbase
+fi
+
+make -j 4
+make install
+cp -r /usr/share/fonts/truetype/dejavu/ $QT_FOLDER/qt5pi/lib/fonts
+tar -zcvf $INIT_FOLDER/qtbase.tar.gz $QT_FOLDER/qt5pi
```
Comment (vpetersson) [inline build_qtbase.sh]: Just write it to the homedir.

File: build_qtbase.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+sudo apt-get update
+sudo apt-get -y build-dep libqt5gui5
+sudo apt-get -y build-dep qt4-x11
+sudo apt-get -y install libudev-dev libinput-dev libts-dev libxcb-xinerama0-dev libxcb-xinerama0
+
+QT_FOLDER=/home/pi/src/qt5
+INIT_FOLDER=$(pwd)
+
+if [ ! -d "$QT_FOLDER" ]; then
+  rm -rf $QT_FOLDER
```
Comment (vpetersson) [inline build_qtbase.sh]: Why do we remove it if it doesn't exist?

File: build_qtbase.sh (hunk 3 above)
Comment (antonmolodykh) [reply build_qtbase.sh]: hmm. Is not it more logical to create an archive in the directory from which the script was called?

Comment (vpetersson) [reply build_qtbase.sh]: No, because that makes it less consistent. You could accidentally fire it off from the wrong directory.

**Comment by vpetersson** [comment]
LGTM

**Pull request merged**